go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/decred/dcrd/database v1.0.2 // indirect
 	github.com/decred/dcrd/dcrec v0.0.0-20180816212643-20eda7ec9229
 	github.com/decred/dcrd/dcrec/edwards v0.0.0-20180816212643-20eda7ec9229 // indirect
-	github.com/decred/dcrd/dcrjson v1.0.0
+	github.com/decred/dcrd/dcrjson v1.0.1-0.20181126011428-6b9e747ca880
```

Comment:
I want to see these dcrd modules published before this goes in, but this is fine for initial testing and review.